lib/ui/views/projects/project_details_view.dart

```diff
@@ -47,6 +48,25 @@ class _ProjectDetailsViewState extends State<ProjectDetailsView> {
     _recievedProject = widget.project;
   }
 
+  void showShareWidget() {
```

Comment:
better to rename this method as `onShareButtonPressed`, just to follow the conventions.